app/Resources/views/pagelayout.html.twig

```diff
@@ -21,6 +22,12 @@
             <link rel="stylesheet" type="text/css" href="{{ asset_url }}">
         {% endstylesheets %}
 
+        {% javascripts
+            'assets/js/prism.js'
```

Comment:
Nope :-) but in the future I'll create a dedicated block for the assets section in the pagelayout file to load assets depending on the full view.